### PR TITLE
CameraWebServer example fix

### PR DESCRIPTION
**Pull request by PilnyTomas**
## Description of Change
Fixed error appearing on verbose core debug level
Fixed warnings (missing initializers)

## Tests scenarios
@P-R-O-C-H-Y tested on ESP32S3-EYE

## Related links
no related issue

### CODE REVIEW

**Review by P-R-O-C-H-Y — APPROVED** [review]
Tested on EPS32S3-eye.
No compilation errors/warnings.
Example working :)
We can merge 👍

**Comment by VojtechBartoska** [comment]
Let's wait for 2nd review from @SuGlider.

**Review by SuGlider — APPROVED** [review]
Nice work @PilnyTomas!

**Pull request merged**
